app/src/main/java/org/oppia/android/app/survey/SurveyWelcomeDialogFragment.kt

```diff
@@ -52,10 +52,9 @@ class SurveyWelcomeDialogFragment : InjectableDialogFragment() {
         }
       }
     }
-  }
-
-  private fun extractQuestions(questionNames: List<SurveyQuestionName>): IntArray {
-    return questionNames.map { questionName -> questionName.number }.toIntArray()
+    private fun extractQuestions(questionNames: List<SurveyQuestionName>): IntArray {
```

Comment:
Nit: newline above this for spacing consistency.

Also, let's update this to just return ``List<Int>``--no need for the conversion to ``IntArray`` and then back to ``List<Int>`` above.

Reply:
Done.